scripts/rabbitmq-server.bat

```diff
@@ -90,9 +90,9 @@ if "!RABBITMQ_IO_THREAD_POOL_SIZE!"=="" {
 +W w ^
 +A "!RABBITMQ_IO_THREAD_POOL_SIZE!" ^
 +P 1048576 ^
--kernel inet_default_connect_options "[{nodelay, true}]" ^
 !RABBITMQ_LISTEN_ARG! ^
 !RABBITMQ_SERVER_ERL_ARGS! ^
+-kernel inet_default_connect_options "[{nodelay, true}]" ^

```

Comment:
I don't see any `+K true` flag for the Windows startup script. Is it on purpose?